packages/lab/src/calendar/internal/CalendarNavigation.tsx

```diff
@@ -274,7 +276,7 @@ export const CalendarNavigation = forwardRef<
         leaveDelay={0} // --salt-duration-instant
       >
         <Button
-          disabled={!canNavigateNext}
+          disabled={!canNavigateNext && !isCompact}
```

Comment:
i don't think you need to pass compact to calendar, could you not just handle this from the picker?

Reply:
removed